client/common/cmdline.c

```diff
@@ -3857,6 +3863,14 @@ BOOL freerdp_client_load_addins(rdpChannels* channels, rdpSettings* settings)
 		}
 	}
 
+	if (settings->RedirectClipboard)
+	{
```

Comment:
Isn't the clipboard channel loaded by the code below when the `RedirectClipboard` setting is set ?

Reply:
@hardening true.